src/main/java/ua/softserveinc/tc/controller/admin/DayOffController.java

```diff
@@ -64,6 +68,7 @@
         if (dayOffService.dayOffExist(dayOff.getName(), dayOff.getStartDate())) {
             log.warn("There is another day off with the same name: " + dayOff.getName() + ", or" +
                     "with the same start date: " + dayOff.getStartDate());
+            return new ResponseEntity<>(HttpStatus.NOT_FOUND);
```

Comment:
Why are you did this? You are prohibiting from create dayoff if there the name or start date are matching. But Why? In the past it only log a warning and successfully create this this dayoff.